rest_api/v2/components/index.yml

```
@@ -133,6 +133,57 @@ schemas:
         $ref: '#/schemas/CategoryName'
       id:
         $ref: '#/schemas/CategoryId'
+  CategorySqlId:
+    type: integer
+    description: Category id
+    example: 42
+  CategorySqlGmbId:
+    type: string
+    description: Google My Business Category Id ( unique )
+    example: 'gcid:climbing_gym'
+  CategorySqlName:
+    type: object
+    properties:
+      name:
+        type: string
+        description: Translated category name
+        example: salle d'escalade
+      country:
+        type: string
+        description: Country code
+        example: 'FR'
+  CategorySqlNames:
+    type: array
+    description: Category name translated for available countries
+    items:
+      $ref: '#/schemas/CategorySqlName'
+  CategorySqlLegacyIds:
+    type: array
+    description: old related partoo categories id
```

To make it consistent with other `description` ;) 
```suggestion
    description: Old related partoo categories id
```

---

👍